### ISSUE TITLE
[TS] yo bot-virtualassistant:skill generated project runtime error

### ISSUE THREAD

**Issue by vkacherov** [microsoft/botframework-solutions#3677]
#### What project is affected?
Yeoman bot-virtualassistant:skill generator

#### What language is this in?
TypeScript

#### What happens?
Runtime error when running 'npm run start'

#### What are the steps to reproduce this issue?
yo bot-virtualassistant:skill
npm install 
npm run start

#### What were you expecting to happen?
bot to run locally

#### Can you share any logs, error output, etc.?
node_modules/adaptive-expressions/lib/functionUtils.d.ts:9:31 - error TS7016: Could not find a declaration file for module '@microsoft/recognizers-text-data-types-timex-expression'. '/Users/vadim/projects/bcg/skills/awesome-skill/node_modules/@microsoft/recognizers-text-data-types-timex-expression/dist/recognizers-text-data-types-timex-expression.umd.js' implicitly has an 'any' type.
  Try `npm install @types/microsoft__recognizers-text-data-types-timex-expression` if it exists or add a new declaration (.d.ts) file containing `declare module '@microsoft/recognizers-text-data-types-timex-expression';`

9 import { TimexProperty } from '@microsoft/recognizers-text-data-types-timex-expression';
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.

#### Any screenshots or additional context?

**Comment by Batta32** [comment]
Thanks @vkacherov for reporting this issue. In order to narrow the problem, we came up with some **questions**:
1. Which version of `generator-bot-virtualassistant` have you installed?
1. Have you checked if the same issue happens generating a Virtual Assistant too?

We will try to reproduce this issue and as soon as we have any update, we will back to you here 😊.

**Comment by vkacherov** [comment]
Hi @Batta32, I'm using the latest version generator-bot-virtualassistant@1.0.1 and seeing the exact same issue generating/running the Virtual Assistant.

I'm running Yeoman 3.1.1, Node v13.12.0 and npm v6.14.8 on MacOS 10.15.6

**Comment by Batta32** [comment]
Thanks @vkacherov for your quick reply, we will try to reproduce the environment and the issue. We will let know you the updates 😊.

**Comment by Batta32** [comment]
Hi @vkacherov, we are currently working on this issue trying to reproduce the environment and the issue following the mentioned steps.

This week we will let you know any update that we can have 😊.

**Comment by Batta32** [comment]
Hi @vkacherov, we successfully ﻿**reproduced** the mentioned issue, and furthermore we have a **workaround**.

We recommend you comment the line `"strict": true` of the [tsconfig.json](https://github.com/microsoft/botframework-solutions/blob/master/templates/typescript/samples/sample-skill/tsconfig.json) file of bot generated, and then run the `npm run start`.

### Additional Context
We raised the same issue #3635 and it's fixed with PR #3631.

In addition, we found the opened issue microsoft/Recognizers-Text#588 explaining that `recognizers-text-data-types-timex-expression` package should have a types file.

Let us know if this helps you 😊.

**Comment by Batta32** [comment]
@peterinnesmsft - we can close the issue due to inactivity. @vkacherov if you are still experiencing issues, feel free to reactivate and we can pick this up again.
